packages/datatrak-web/src/Routes.tsx

```diff
@@ -95,10 +95,12 @@ export const Routes = () => {
             </Route>
             <Route path={ROUTES.SURVEY} element={<SurveyPage />}>
               <Route index element={<SurveyStartRedirect />} />
-              <Route path={ROUTES.SURVEY_REVIEW} element={<SurveyReviewScreen />} />
               <Route path={ROUTES.SURVEY_SUCCESS} element={<SurveySuccessScreen />} />
-              <Route path={ROUTES.SURVEY_SCREEN} element={<SurveyScreen />} />
-              <Route path={ROUTES.QUESTIONS} element={<SurveyQuestionsPage />} />
+              <Route element={<ScrollableLayout />}>
```

Comment:
I think that the layouts seem a bit too nested here and it is a bit hard to follow and maintain. Do you think that it's possible to remove one layer of nesting by refactoring the SurveyPage and ScrollableLayout?

Reply:
I wondered that, and sort of did something similar in the review screen PR, so will do that as part of that one